src/Adapter/MailTemplate/MailPreviewVariablesBuilder.php

```diff
@@ -146,6 +148,20 @@ public function buildTemplateVariables(LayoutInterface $mailLayout)
         $templateVars['{color}'] = $this->configuration->get('PS_MAIL_COLOR');
         $templateVars = array_merge($templateVars, $this->buildOrderVariables($mailLayout));
 
+        if ($mailLayout->getModuleName() !== '') {
```

Comment:
I am not sure to understand why this check

Reply:
It's to allow to add only related vars to the specific layout coming from a module ; it's not needed to add specific layout var - as I think - for another layours than the module itself.